app/src/main/java/at/roteskreuz/stopcorona/model/repositories/ReportingRepository.kt

```diff
@@ -94,7 +94,7 @@ interface ReportingRepository {
     /**
      * Observe the personal data.
      */
-    fun observePersonalData(): Observable<PersonalData>
+    fun observePersonalData(): Observable<PhoneNumberData>
```

Comment:
If you are renaming it, you should do it in all places.
For me, it's completely ok to call it PersonalData where we have only one field. It's more general. But if you want to rename it, please do it in all places appropriately.